### PR TITLE
Kamu UI 294 support open in new tabwindow shortcuts

### PR DESCRIPTION
**Pull request by dmitriy-borzenko**
Closes: https://github.com/kamu-data/kamu-web-ui/issues/294

### CODE REVIEW

File: src/app/app.component.spec.ts [hunk 1]
```diff
@@ -44,15 +45,23 @@ describe("AppComponent", () => {
...
             ],
-            declarations: [AppComponent, ModalComponent, AppHeaderComponent, SpinnerComponent],
             providers: [
                 SearchService,
                 SearchApi,
                 AuthApi,
                 NavigationService,
                 ModalService,
-                { provide: Router, useValue: routerMock },
+                // { provide: Router, useValue: routerMock },
```
Comment (zaychenko-sergei) [inline src/app/app.component.spec.ts:64]: remove?

File: src/app/components/app-header/app-header.component.html [hunk 2]
```diff
@@ -89,14 +89,14 @@
             aria-label="Global"
         >
             <div class="d-flex flex-column flex-md-row flex-self-stretch flex-md-self-auto">
-                <a
+                <!-- <a
```
Comment (zaychenko-sergei) [inline src/app/components/app-header/app-header.component.html:92]: Remove?

File: src/app/app.component.spec.ts (hunk 1 above)
Comment (dmitriy-borzenko) [reply src/app/app.component.spec.ts:64]: Removed

File: src/app/components/app-header/app-header.component.html (hunk 2 above)
Comment (dmitriy-borzenko) [reply src/app/components/app-header/app-header.component.html:92]: removed

**Review by zaychenko-sergei — APPROVED** [review]

**Pull request merged**